sdk/src/types/block/unlock/mod.rs

```diff
@@ -34,10 +34,12 @@ pub(crate) type UnlockIndex = BoundedU16<{ *UNLOCK_INDEX_RANGE.start() }, { *UNL
 #[derive(Clone, Eq, PartialEq, Hash, From, Packable)]
 #[packable(unpack_error = Error)]
 #[packable(tag_type = u8, with_error = Error::InvalidUnlockKind)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize), serde(untagged))]
 pub enum Unlock {
     /// A signature unlock.
     #[packable(tag = SignatureUnlock::KIND)]
-    Signature(SignatureUnlock),
+    #[from(ignore)]
```

Comment:
Can you use `#[from(forward)]` here?

Reply:
Probably! I'll try it

Reply:
Turns out, I cannot. It causes the other impls to break.